rep-2000.rst

```diff
@@ -547,6 +547,126 @@ Build System Support:
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| EmPY        |                                      3.3.2                                             |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Gazebo      |     11.0.0     |     11.0.0    |      N/A       |      11.0.0    |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ignition    |             Citadel            |      N/A       |     Citadel    |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ogre        |                                      1.10*                       |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenCV      |     4.2.0      |     4.2.0     |     3.4.6*     |      3.2.0     |  4.1.0 / 3.2.0****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenSSL     |     1.1.1c     |     1.0.2r    |     1.0.2r     |      1.1.1c    | 1.1.1d / 1.1.1b**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.4        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Python      |     3.8.0      |     3.7.3     |     3.8.0      |      3.7.3     |  3.8.2 / 3.7.5****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Qt          |     5.12.5     |     5.12.3    |     5.10.0     |      5.11.3    | 5.14.1 / 5.12.5**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                              |         **Linux only**                                                |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| PCL         |     1.10.0     |     N/A       |     N/A        |      1.9.1     |        1.8.1        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                                **RMW DDS Milddleware Providers**                                     |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Connext DDS |                              5.3.1***           |               N/A                    |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Cyclone DDS |                                 0.5.0 (Eusebius)                                       |
```

Comment:
This should probably be 0.5.1?

Reply:
Looks like that is the most recent release. 90b959d